django/core/management/commands/loaddata.py

```diff
@@ -288,6 +299,13 @@ def parse_name(self, fixture_name):
         """
         Split fixture name in name, serialization format, compression format.
         """
+        if fixture_name == '-':
+            if not self.format:
+                raise CommandError(
+                    "Must specify format when reading from stdin")
```

Comment:
@charettes 
Do we have convention for preferring single quotes? If so, I guess it should be added to the coding style.